BrainPortal/db/schema.rb

```diff
@@ -117,6 +122,35 @@
   add_index "data_providers", ["type"], :name => "index_data_providers_on_type"
   add_index "data_providers", ["user_id"], :name => "index_data_providers_on_user_id"
 
+  create_table "demands", :force => true do |t|
```

Comment:
That table is also spurious! Check you dev DB, it seems you've left it in an inconsistent state, so the schema here is reconstructed wrong.